charts/trino/values.yaml

```diff
@@ -247,6 +254,17 @@ worker:
     # failureThreshold: 6
     # successThreshold: 1
 
+  lifecycle: {}
+    # preStop:
+    #   exec:
+    #     command: ["/bin/sh", "-c", "curl -v -X PUT -d '\"SHUTTING_DOWN\"' -H \"Content-type: application/json\" http://localhost:8081/v1/info/state"]
+    # https://trino.io/docs/current/admin/graceful-shutdown.html#admin-graceful-shutdown--page-root
+    # access control does not allow graceful shutdowns by default, to configure check the accessControl section
+
+  terminationGracePeriodSeconds: 30
+  # The terminationGracePeriodSeconds should be greater or equal to shutdown.grace-period.
+  # To configure shutdown.grace-period, check the lifecycle section.
```

Comment:
The lifecycle section doesn't explain how to configure `shutdown.grace-period` using this chart.

Reply:
I rewrite the `lifecycle` comment again, I hope now it's more clear about.

Reply:
It's better, but it still doesn't say how to configure `shutdown.grace-period` using this chart.

Also, I just noticed that `shutdown.grace-period` defaults to 2 minutes. `terminationGracePeriodSeconds` defaults to 30s in Kubernetes, so it's a good value to preserve backward compatibility. It would be better to move this comment to the lifecycle section.

Reply:
Oh, I understand your question now. 
Trino documentation doesn't mention how to change the default value of `shutdown.grace-period`, but searching in [Trino source code](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/java/io/trino/server/ServerConfig.java#L76-L81), I noticed that could be changed on `config.properties`.

Since it's not in the Trino documentation, I don't know the implications of changing this default value.

Reply:
Yeah, sorry for not being explicit from the beginning.

`shutdown.grace-period` should be added to `config.properties`, which is generated in this chart in a pretty complex way: https://github.com/trinodb/charts/blob/main/charts/trino/templates/configmap-coordinator.yaml#L46

Looks like it's best to add it in `additionalConfigProperties` that's used for all nodes.

Reply:
How about now?

Reply:
Much better, all the required info is there, thanks!